packages/fxa-settings/src/components/Settings/FlowRecoveryKeyHint/en.ftl

```diff
@@ -0,0 +1,12 @@
+## Fourth view in the Recovery-Key-Add UX redesign flow
+
+# This will be the title of the back button so that users know where that button will take them
+flow-recovery-key-hint-back-button-title = Back to settings
```

Comment:
This ftl string will be passed in from `PageRecoveryKeyCreate` because it is reused in all flow pages.